### ISSUE TITLE
Accessibility: Screen Reader Visibility

### ISSUE THREAD

**Issue by MountainCedar** [rich00feld/RestoratiON#69]
I've gone through the app using [shinya11y](https://github.com/ewenme/shinya11y) to look at screen reader visibility, and the good news is that most of the text in the app is at least visible to a screen reader currently (including tables), with some exceptions:

- Any input text boxes (e.g. for setting the buffer value, picking the number of principal components etc.) are not detected. I'm not sure how to rectify this, but I'll look into seeing if they can be coded differently using HTML. We talked about changing sliders to text input so this is essential if we plan to go this route. 
- Plots, and any text in plots (titles, legends etc.) is not visible. This does not include the leaflet map, which has a UI that is visible to screen reader, but polygons that are not. 

Aside from this, I think app text could organized to be better read by a screen reader. Right now sometimes sub-headers appear both in the main text body and on their own and I would think this would be confusing as this means they are duplicated:

![screenreader1](https://github.com/user-attachments/assets/b035a396-5e3d-4fdd-9336-df8fbc973cb9)
